src/tools/clippy/tests/ui/match_overlapping_arm.stderr

```diff
@@ -59,5 +59,24 @@ note: overlaps with this
 LL |         0..=10 => println!("0 ... 10"),
    |         ^^^^^^
 
-error: aborting due to 5 previous errors
+error: this `if` has identical blocks
```

Comment:
This must not lint.

Reply:
They have the same pattern, expression and block return type so why this snippet shouldn't lint? If desirable, I can transfer the code to `if_same_then_else.rs`

Reply:
Oh wait, I thought this test was added because it tests that the comments are different. I'm now noticing that this test file shouldn't even test `if_same_then_else`. Can you just add an `#[allow(clippy::if_same_then_else)` to this test file and bless it again, please?

Reply:
Done